src/Microsoft.AspNetCore.Mvc.Core/Infrastructure/MvcCompatibilityOptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.Infrastructure
+{
+    public class MvcCompatibilityOptions
+    {
+        public CompatibilityVersion CompatibilityVersion { get; set; } = CompatibilityVersion.Version_2_0;
```

Comment:
I guess this default value will remain untouched until 3.0, right?

Reply:
If this is true, we should log a bug for the 3.0 milestone to update this value. Otherwise you know we'll forget.

Reply:
👍 Yep. The idea is also that we're drop support for all of the *old* behaviors that we supported in various 2.X release. So the work is more significant than just updating the default, we'd probably obsolete all of the old enum values - I'm not sure yet if its better to remove old enum values or just obsolete them.

Reply:
Ah yes good point! I think we leave them but obsolete them so that we can alert the developer. Otherwise they won't know. Hopefully most people won't even use the switches too long after they migrate.